misc/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
Please add meaning full comments, for example:
Creating a base model for common fields.